fun error() {
  return nil + false;
}

error.call();